scripts/apps/search/components/ItemList.jsx

```diff
@@ -102,7 +102,10 @@ export class ItemList extends React.Component {
             positionStart = _.indexOf(this.state.itemsList, this.state.selected);
         }
 
-        for (var i = positionStart; i <= positionEnd; i++) {
+        var start = positionStart < positionEnd ? positionStart : positionEnd,
```

Comment:
this could be also like:
```
var start = Math.min(positionStart, positionEnd),
     end = Math.max(positionStart, positionEnd);
```
not sure if it's more readable or not?